README.md

```diff
@@ -29,32 +29,33 @@ the configuration file.
 
 Credentials for accessing Logilica are provided by command line options or by
 the environment variables, `LOGILICA_DOMAIN`, `LOGILICA_EMAIL`, and
-`LOGILICA_PASSWORD`.  (Note to the Developer Practices Team:  the appropriate
+`LOGILICA_PASSWORD`. (Note to the Developer Practices Team: the appropriate
 values for the bot accounts can be obtained from the Bitwarden vault; users
 with admin privileges can also create their own username/password credentials
-using the Logilica UI.)  For interactive use, OAuth or SSO credentials can be
+using the Logilica UI.) For interactive use, OAuth or SSO credentials can be
 used. The tool normally runs in "headless" mode; however, in order to enable
 the user to perform an SSO login, the tool opens a browser window during the
-run.  (You should not interact with this window unless it prompts for your SSO
+run. (You should not interact with this window unless it prompts for your SSO
 credentials; otherwise, you are likely to disrupt the functioning of the tool.)
 
 To run the tool, check out the Git repo and run:
-```pip install -r requirements.txt .```
+`pip install -r requirements.txt .`
```

Comment:
I prefer the fenced-code presentation:  I think it is easier to read and use, and it is often rendered with a "copy" button which makes transferring it to a terminal trivial.

Ditto line 47.

Reply:
I wonder why this change is attributed to me but I have the same preference, so I will update it.